network-api/networkapi/wagtailpages/migrations/0030_populate_profile_slug.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Kevin on 2022-06-17 09:55
+
+import django.db.models.deletion
+import modelcluster.fields
+from django.db import migrations, models
+from networkapi.wagtailpages.models import Profile
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailpages', '0029_adds_slug_field'),
+    ]
+
+    def populate_profile_slugs(apps, schema_editor):
+        """Call save on the Profile snippets to populate
+        slug values"""
+        for profile in Profile.objects.all():
+            profile.save()
+
+    operations = [
+        migrations.RunPython(populate_profile_slugs),
```

Comment:
**Suggestion** I believe we should be passing `RunPython.noop` as the `reverse_code` to allow this migration to be reversed. Otherwise, I think, backward migrations will fail at this step. 

https://docs.djangoproject.com/en/4.0/ref/migration-operations/#django.db.migrations.operations.RunPython.noop